lower_obst: [1,0,60]
upper_obst: [30,255,255]
